docs/SPEC.md

```diff
@@ -1081,7 +1078,8 @@ Every statement contained in an imported package is evaluated.
 
 ### Attributes
 
-Attributes define a set of properties on the subsequent source code element.
+Attributes define a set of properties on source code elements.
+An outer attribute applies to the outer context it is found in, while an inner attribute applies to the subsequent context.
```

Comment:
I thought the purpose of this change would be to remove the distinction of inner vs. outer attributes. Hence attributes always requiring a subsequent element to attach to. Am I missing something?

Reply:
This line was left in during the split, I'll remove/reword it.